Document key kind restrictions for Secure Session (#18)
cossacklabs/product-docs: themis/crypto-theory/cryptosystems/secure-session.md

and it must uniquely identify the public key used by a peer.

**Keypairs** can be securely generated with Themis key generation functions.

> Secure Session only supports EC keys. RSA support may be added in the future.

Reviewer: ```suggestion
Secure Session only supports EC keys. RSA support is available per request only.
```

> Please [contact us](mailto:dev@cossacklabs.com) if you need to use a different key type.

Reviewer: ```suggestion
Please [contact us](mailto:dev@cossacklabs.com) if you need to use RSA or other key types.
```

    size_t*  private_key_length,
    uint8_t* public_key,
    size_t*  public_key_length);

themis_status_t themis_gen_rsa_key_pair(
    uint8_t* private_key,
    size_t*  private_key_length,
    uint8_t* public_key,

>     size_t*  public_key_length);

Reviewer 2: Nice attention to detail :)
